assets/src/blocks/Spreadsheet/SpreadsheetBlock.js

```diff
@@ -1,26 +1,46 @@
-import {SpreadsheetEditor} from './SpreadsheetEditorScript';
+import {SpreadsheetEditor} from './SpreadsheetEditor';
 import {CSS_VARIABLES_ATTRIBUTE} from '../CssVariablesAttribute';
 import {frontendRendered} from '../frontendRendered';
+import {SpreadsheetFrontend} from './SpreadsheetFrontend';
+import {renderToString} from 'react-dom/server';
 
 const BLOCK_NAME = 'planet4-blocks/spreadsheet';
 
+const attributes = {
```

Comment:
What do you think if we define the next attribute by default?
```javascript
{
      url: {
    type: 'string',
    default: '',
  },
}
```
and then override it.

We'll avoid to repeat the `url` field in both versions. For instance,
```javascript
{
   ...attributes,
    color: {
         type: 'string',
         default: 'grey',
     },
}
```
OR
```javascript
```javascript
{
   ...attributes,
    css_variables: CSS_VARIABLES_ATTRIBUTE,
}
```
```

Reply:
If we do it this way then this version: 
```
{ ...attributes, color: { type: 'string', default: 'grey', }, }
```
will be duplicated (once for the actual attributes, and once in the `deprecated` object) so I'm not sure it's better than the current implementation?